cheeseshop/dbapi.py

```diff
@@ -311,10 +311,79 @@ class CsGoSteamId(object):
         await conn.execute('''
             CREATE TABLE cs_go_steam_ids(
                 id serial PRIMARY KEY,
-                steam_id text UNIQUE NOT NULL
+                steam_id text UNIQUE NOT NULL,
+                print_name text,
+                team integer REFERENCES cs_go_team_ids (id)
             )
         ''')
 
+    @staticmethod
+    async def get_all(conn):
+        players = []
+        async for record in conn.cursor('''
+            SELECT * from cs_go_steam_ids
+        '''):
+            players.append({
+                'id': record['steam_id'],
+                'name': record['print_name'],
+                'team': record['team_name}']})
+        return players
+
+
+class CsGoTeamNames(object):
+    @staticmethod
+    async def create_schema(conn):
+        await conn.execute('''
+            CREATE TABLE cs_go_team_names(
```

Comment:
For now we can probably remove this table in the name of simplicity and add a column in the main teams table called hltv_name which is probably what well want to use for team names.